### PR TITLE
Bring license in line with FIRST's

### PR DESCRIPTION
**Pull request by Muirrum**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,29 @@
-MIT License
+BSD 3-Clause License
 
-Copyright (c) 2018 AndromedaOne
+Copyright (c) 2009-2018, FIRST
```
Comment (themccubbins) [inline LICENSE:26]: It won't be (c) FIRST, it will still be (c) AndromedaOne. And Might as well change it to 2018-2019 also.

**Comment by themccubbins** [comment]
So, since this repo is already public, changing the license isn't as simple as pushing a change to the repo's LICENSE file....there are legal implications. I'm not a lawyer but since there are many contributors it may be necessary to get +1 's from every single contributor for this to be valid. Also even if it was modified, all previous commits to the repo will still be considered as under the MIT license anyway.

So:

a) if we really want to add BSD to this repo, we should consider dual licensing it since it pretty much would be dual licensed already.
b) why do we need to do this? Is there a requirement from FIRST for a particular license? Just because FIRST uses BSD doesn't mean we have to.

Suggestion: leave this repo as MIT and don't bother with adding BSD. Potentially switch to BSD next season.

**Review by themccubbins — CHANGES_REQUESTED** [review]

**Comment by themccubbins** [comment]
PS Welcome to the complicated world of licenses.

**Comment by Muirrum** [comment]
I'd disagree. There isn't much support for MIT license, WPILib is under BSD 3-Clause, and part of that license is the requirement that the BSD license be redistributed with any derivatives.

**Comment by themccubbins** [comment]
Disagree with which part of what I said?

This code isn't a derivative work. Unless I missed a class where we copy pasted from something in WPILib.

**Comment by themccubbins** [comment]
RobotBuilder Generated code may be covered under their BSD license...this is a grey area IMO (but again I'm not a lawyer). In which case, yeah, we should probably at least include a copy of their license as per the terms of their license.

**Comment by Muirrum** [comment]
Aye, that sounds like the best bet. @themccubbins I've put the BSD license below the MIT license.

**Pull request closed without merging**